Update words.md - stilistische Änderungsvorschläge L690 - 720 (Ende) (#34)
TQueV/covid-19: words/words.md

    </div>
</div>

Das Flugzeug ist untergegangen. Wir hocken zusammengedrängt in den Rettungsbooten. Es ist Zeit, Festland zu suchen.[^dry_land]

[^dry_land]: Festland-Metapher [von Marc Lipsitch und Yonatan Grad, auf STAT News](https://www.statnews.com/2020/04/01/navigating-covid-19-pandemic/)

> Teams aus Epidemiologinnen und Epidemiologen und Menschen in der Politik ([links](https://www.americanprogress.org/issues/healthcare/news/2020/04/03/482613/national-state-plan-end-coronavirus-crisis/), [rechts](https://www.aei.org/research-products/report/national-coronavirus-response-a-road-map-to-reopening/ ), und [überparteilich](https://ethics.harvard.edu/covid-roadmap)) haben einen Konsens darüber erzielt, wie wir COVID-19 schlagen können und gleichzeitig unser Leben *und* unsere Freiheiten schützen.

Reviewer: ```suggestion
Teams aus Epidemiologinnen und Epidemiologen und Menschen in der Politik ([progressiv](https://www.americanprogress.org/issues/healthcare/news/2020/04/03/482613/national-state-plan-end-coronavirus-crisis/), [konservativ](https://www.aei.org/research-products/report/national-coronavirus-response-a-road-map-to-reopening/ ), und [überparteilich](https://ethics.harvard.edu/covid-roadmap)) haben einen Konsens darüber erzielt, wie wir COVID-19 schlagen können und gleichzeitig unser Leben *und* unsere Freiheiten schützen.
```

Reviewer: Hier die zugehörige Änderung

    </div>
</div>

> Das Flugzeug ist versunken. Wir hocken zusammengedrängt in den Rettungsbooten. Es ist Zeit, Festland zu suchen.[^dry_land]

Reviewer 2: ```suggestion
Das Flugzeug ist untergegangen. Wir hocken zusammengedrängt in den Rettungsbooten. Es ist Zeit, Festland zu suchen.[^dry_land]
```
Denke das wäre die üblichere Wortwahl, oder?

[^dry_land]: Festland-Metapher [von Marc Lipsitch und Yonatan Grad, auf STAT News](https://www.statnews.com/2020/04/01/navigating-covid-19-pandemic/)

> Teams aus Epidemiologinnen und Epidemiologen und Menschen in der Politik ([links](https://www.americanprogress.org/issues/healthcare/news/2020/04/03/482613/national-state-plan-end-coronavirus-crisis/), [rechts](https://www.aei.org/research-products/report/national-coronavirus-response-a-road-map-to-reopening/ ), und [überparteilich](https://ethics.harvard.edu/covid-roadmap)) haben einen Konsens darüber erzielt, wie wir COVID-19 schlagen können und gleichzeitig unser Leben *und* unsere Freiheiten schützen.

Reviewer 2: In den USA ist das politische Gefüge in der Politik recht anders als bei uns. Kann jemand was dazu sagen ob wir hier links/rechts einfach so direkt übersetzt übernehmen können? Die etablierten Parteien in den USA sind aus unserer Sicht alle eher konservativ. Die verlinkten Artikel könnte ich jetzt politisch nicht so einfach einordnen (bzw. müsste sie vorher lesen). Evtl. sehr wortreich: "im linken Spektrum der US-Politik" / "im rechten Spektrum der US-Politik"?

Reviewer 2: Oder evtl. einfach "links der Mitte", "rechts der Mitte"? "Links" und "rechts" verwenden wir gefühlt eher für die extremeren Linken und Rechten. Und "Die Rechten" sind ja nur Nazis oder nah dran?

Reviewer: DIe Organisationen, die hinter den verlinkten Artikeln stehen, lassen sich meiner kurzen Recherche nach sehr eindeutig als "progressiv", "konservativ" und "überparteilich" bezeichnen.

> Hier ist die grobe Idee, mit einigen (weniger konsensfähigen) Backup-Plänen:

Reviewer 2: Ganz am Anfang haben wir schon Mal eine Stelle gehabt wo dann "backup plans" als "Plan B" übersetzt wurde. Das passt dann jetzt hiermit nicht zusammen. Die Option mit "Plan B" geht hier aber nicht, weil der Plural hier zwingend ist.

Reviewer: Das stimmt, er ist hier zwingend. Also entweder wir sind konsistent und nehmen am Anfang und am Schluss den Plural oder wir sind inkonsistent und nehmen oben den Singular ("Plan" und "Plan B") und hier, Vorschlag: "Alternativplänen"

Ich tendiere, aus Gründen der Lesbarkeit beim Einstieg in den Text (das finde ich wichtig, weil ich glaube, dass so mehr Menschen zum Weiterlesen animiert werden, die mit dem Thema in der Tiefe nicht so viel zu tun haben und mal schnell unterhalten werden wollen), zu Variante 2.

Hier ist die grobe Idee, mit einigen (weniger konsensfähigen) Backup-Plänen:

![](pics/plan.png)

Was bedeutet das für DICH, jetzt gerade?

> **Für jeden:** Respektiere die Ausgangsbeschränkungen, damit wir so schnell wie möglich aus Phase I herauskommen können. Wasche dir weiter die Hände. Stelle deine eigenen Masken her. Lade eine *privatsphäreschützende* App zur Ermittlung von Kontaktpersonen herunter, sobald diese im nächsten Monat verfügbar ist. Bleib gesund - körperlich und geistig! Und schreib den lokalen Entscheidungsträgern, dass sie ihren Hintern hochbewegen sollen und...

Reviewer: Vorschlag: Analog zu unten auch hier "datenschutzfreundliche App"

> **Für Politikerinnen und Politiker:** Machen Sie Gesetze zur Unterstützung von Menschen, die sich selbst isolieren/quarantänisieren müssen. Stellen Sie mehr menschliche Kontaktverfolgerinnen und Kontaktverfolger ein, *unterstützt* durch datenschutzfreundliche Apps. Leiten Sie mehr Mittel in die Dinge, die wir aufbauen sollten, wie..

Reviewer 2: ```suggestion
**Für Politikerinnen und Politiker:** Machen Sie Gesetze zur Unterstützung von Menschen, die sich selbst isolieren/in Quarantäne begeben müssen. Stellen Sie mehr Personal zur manuellen Kontaktverfolgung ein, *unterstützt* durch datenschutzfreundliche Apps. Leiten Sie mehr Mittel in die Dinge, die wir aufbauen sollten, wie..
```

Reviewer: Vorschlag: "Erlassen" statt "Machen"

Reviewer: Außerdem würde ich "aufbauen" analog zu oben bzw. unten ändern, mein Favorit wäre "herstellen"

> **Für Machende:** Baut Tests. Baut Beatmungsgeräte. Baut Personenschutzausrüstung für Krankenhäuser und Praxen. Baut Tests. Baut Masken. Baut Apps. Baut antivirale Medikamente, Prophylaktika und andere Behandlungen, die keine Impfstoffe sind. Baut Impfstoffe. Baut Tests. Baut Tests. Baut Tests. Baut Hoffnung.

Reviewer 3: Ich denke, "Maker" sollte nicht ins Deutsche übersetzt werden, weil das absolut unüblich ist. Aller Personen, die sich davon angesprochen fühlen sollen, werden den Begriff "Maker" kennen, und falls Zweifel bestehen, dass andere ihn nicht kennen, könnte man ggf. [eine Erklärung](https://de.wikipedia.org/wiki/Maker) verlinken.

Author: Ja, Lena, stimme ich dir zu.

Reviewer 2: Sehe ich auch so.

Reviewer: Wie wäre es mit "Herstellerinnen und Hersteller"? Schließlich geht es um Dinge die hergestellt werden und der angesprochene Personenkreis würde sich wohl auch mehrheitlch selbst so nennen

Reviewer: Außerdem und in diesem Zusammenhang: "Behandlungsmethoden" statt "Behandlungen"?

> **Für Machende:** Baut Tests. Baut Beatmungsgeräte. Baut Personenschutzausrüstung für Krankenhäuser und Praxen. Baut Tests. Baut Masken. Baut Apps. Baut antivirale Medikamente, Prophylaktika und andere Behandlungen, die keine Impfstoffe sind. Baut Impfstoffe. Baut Tests. Baut Tests. Baut Tests. Baut Hoffnung.

Reviewer 2: Ich hätte hier auch gerne durchgängig das gleiche Verb, aber spätestens bei "Baut Medikamente" funktioniert das nicht mehr.

"Produziert" ginge durchgängig klingt aber mehr nach Industrie als nach Makern. "Erstellt" funktioniert nicht perfekt (zB mit "Hoffnung" nicht), wäre aber evtl.  ein Kompromiss. "Stellt … her" passt überall, ist aber auch nur nen Synonym für "produziert" und umständlich. Ich tendiere gerade zu "macht", weil es dann schön auf "macht Hoffnung" endet. Daher mein bester Vorschlag:
```suggestion
**Für Machende:** Macht Tests. Macht Beatmungsgeräte. Macht Personenschutzausrüstung für Krankenhäuser und Praxen. Macht Tests. Macht Masken. Macht Apps. Macht antivirale Medikamente, Prophylaktika und andere Behandlungen, die keine Impfstoffe sind. Macht Impfstoffe. Macht Tests. Macht Tests. Macht Tests. Macht Hoffnung.
```

Reviewer: Das ist wirklich ein sehr charmanter Schluss! Ich tendiere, analog zu oben, zu "Stellt ... her", da ich "Macht" sprachlich etwas zu generisch finde.

Das Industrie-Argument verstehe ich, aber ich finde dass "Herstellung" im Gegensatz zu "Produktion" keine unbedingte Assoziation mit Industrie-Scales hat. Zudem handelt es sich bei den meisten hier aufgelisteten Sachen um Dinge, die industriell gefertigt werden. Bzw. wenn nicht, dann vll. mit dem 3D-Drucker. Und auch dann handelt es sich um eine "Produktion" bzw. "Herstellung".
